integration-commons/src/main/java/org/symphonyoss/integration/exception/ExceptionMessageFormatter.java

```diff
@@ -79,12 +77,31 @@ public static String format(String component, String message,
     return getMessage(component, message, solutions, null);
   }
 
+
+  /**
+   * Formats the message following this sample:
+   *
+   * Component: <Component>
+   * Message: <Message>
+   * Solutions:
+   * <Solution A>
+   * @param component The component where the exception was thrown.
+   * @param message The message why the exceptions happened.
+   * @param solution The solution provided to the exceptions.
+   * @return Formatted message
+   */
+  public static String format(String component, String message,
+      String solution) {
+    List<String> solutions = new ArrayList<>();
+    solutions.add(solution);
+    return getMessage(component, message, solutions, null);
```

Comment:
Could we use varargs parameters insteadof a list of strings?

Something like that: getMessage(String component, String message, Throwable t, String... solutions)